web_ui/src/pages/media/hooks/media-delete/media-delete.hook.ts

```diff
@@ -17,6 +18,8 @@ interface UseDeleteMediaMutation {
     deleteMedia: UseMutationResult<unknown, AxiosError, MediaItem[]>;
 }
 
+const BATCH_SIZE = 300;
```

Comment:
300 feels like a bit big, perhaps it would be better to lower it to say 10 so that we also don't bombard the server with this many requests at once?

Reply:
Agree, 300 feels a bit too much, 10 or 20 should suffice

Reply:
> 300 feels like a bit big, perhaps it would be better to lower it to say 10 so that we also don't bombard the server with this many requests at once?

There were about 6 concurrent requests in the queue. The main problem was that scheduling a few hundred requests failed with `net::ERR_INSUFFICIENT_RESOURCES'. So it's a browser-side fix only.